kibbeh/src/modules/user/UserPage.tsx

```diff
@@ -11,6 +12,11 @@ export const UserPage: PageComponent<UserPageProps> = ({}) => {
   const { query } = useRouter();
   return (
     <DefaultDesktopLayout>
+      <HeaderController
+        title={
+          query.username && !Array.isArray(query.username) ? query.username : ""
+        }
+      />
       <MiddlePanel>
         <UserProfileController
           key={typeof query.username === "string" ? query.username : ""}
```

Comment:
Why not use this same logic? perhaps, putting it in a `const` so you do not have to write it twice!?

Reply:
Oh snap, my mind fried and didn't see it, thanks I will change it!